cupy/core/_kernel.pxd

```diff
@@ -35,6 +35,15 @@ cdef class _ArgInfo:
         readonly bint c_contiguous
         readonly bint index_32_bits
 
+    cdef _ArgInfo _init(
+            self,
+            _ArgKind arg_kind,
+            type typ,
+            object dtype,
+            int ndim,
+            bint c_contiguous,
+            bint index_32_bits)
```

Comment:
Could you fix indentation for Travis failure?

Reply:
I fixed.